core/src/main/resources/error/error-classes.json

```diff
@@ -1198,6 +1198,11 @@
       "Invalid SQL syntax:"
     ],
     "subClass" : {
+      "ANALYZE_TABLE_UNEXPECTED_NOSCAN" : {
+        "message" : [
+          "ANALYZE TABLE ... COMPUTE STATISTICS ... <ctx> must be 'NOSCAN'"
```

Comment:
1. This error can come from slightly other command: `ANALYZE TABLES IN default COMPUTE STATISTICS`
2. Let's follow `toSQLStmt`, and don't wrap NOSCAN.

Reply:
ANALYZE TABLE ... COMPUTE STATISTICS ... <ctx> must be 'NOSCAN'
->
ANALYZE TABLE(S) ... COMPUTE STATISTICS ... <ctx> must be either NOSCAN or empty